.circleci/images/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM circleci/php:7.4-fpm-node-browsers
+
+# install system packages
+RUN sudo apt-get install -qqy libzip-dev libpng-dev libjpeg-dev libfreetype6-dev
```

Comment:
what about a single `USER root` statment before the block, that a `USER circleci` at the end?

Reply:
make sense! ;)